plugin/evm/atomic/sync/syncer_test.go

```diff
@@ -182,39 +182,37 @@ func TestSyncerParallelizationScenarios(t *testing.T) {
 		description       string
 	}{
 		{
-			name:              "parallelization with 4 workers",
-			targetHeight:      2 * uint64(testCommitInterval), // 2,048 leaves for meaningful parallelization
-			numWorkers:        4,
-			useDefaultWorkers: false,
-			description:       "should work correctly with 4 worker goroutines",
+			name:         "parallelization with 4 workers",
+			targetHeight: 2 * uint64(testCommitInterval), // 2,048 leaves for meaningful parallelization
+			numWorkers:   4,
+			description:  "should work correctly with 4 worker goroutines",
 		},
 		{
-			name:              "default parallelization",
-			targetHeight:      uint64(testCommitInterval), // 1,024 leaves to test commit boundary
-			numWorkers:        0,
-			useDefaultWorkers: true,
-			description:       "should default to parallelization with default workers",
+			name:         "default parallelization",
+			targetHeight: uint64(testCommitInterval), // 1,024 leaves to test commit boundary
+			numWorkers:   0,
+			description:  "should default to parallelization with default workers",
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			ctx, mockClient, atomicBackend, clientDB, root := setupParallelizationTest(t, tt.targetHeight)
-			runParallelizationTest(t, ctx, mockClient, clientDB, atomicBackend, root, tt.targetHeight, tt.numWorkers, tt.useDefaultWorkers)
+			runParallelizationTest(t, ctx, mockClient, clientDB, atomicBackend, root, tt.targetHeight, tt.numWorkers)
 		})
 	}
 }
 
 // TestSyncerContextCancellation verifies that the syncer properly handles context cancellation.
 func TestSyncerContextCancellation(t *testing.T) {
 	ctx, mockClient, atomicBackend, clientDB, root := setupParallelizationTest(t, testTargetHeight)
-	config := createTestConfig(testTargetHeight)
-	syncer := createTestSyncer(t, mockClient, clientDB, atomicBackend, root, config)
+	syncer, err := NewSyncer(mockClient, clientDB, atomicBackend.AtomicTrie(), root, testTargetHeight)
+	require.NoError(t, err, "could not create test syncer")
```

Comment:
```suggestion
	require.NoError(t, err, "NewSyncer()")
```

Rationale: https://google.github.io/styleguide/go/decisions#useful-test-failures and specifically https://google.github.io/styleguide/go/decisions#identify-the-function